enterprise/admin/views.py

```diff
@@ -241,20 +244,23 @@ def get_search_keyword(self, request):
         """
         return request.GET.get('q', None)
 
-    def get_enterprise_customer_user_queryset(self, search_keyword, customer_uuid):
+    def get_enterprise_customer_user_queryset(self, request, search_keyword, customer_uuid, page_size=10):
```

Comment:
I think default `page_size` should be `25`. It would be nice if we can store it as constant.